docker-compose.yml

```diff
@@ -4,7 +4,12 @@ services:
     build: .
     container_name: outliers_dev
     command: "python3 outliers.py interactive --config /mappedvolumes/config/outliers.conf --use-cases /use_cases/*.conf"
+#    environment:
+#      - es_username=outliers
+#      - es_password=outliersw
```

Comment:
Why outliers and outliersw?
I would have set a more explicit username and password like my_es_username and my_es_password